### PR TITLE
Ericbrehault/sc 4276/publish our svelte component on npm js

### PR DESCRIPTION
**Pull request by ebrehault**

### CODE REVIEW

File: libs/search-widget/package.json [hunk 1]
```diff
@@ -1,4 +1,33 @@
 {
   "name": "@nuclia/widget",
-  "version": "0.0.1"
+  "version": "1.0.0",
+  "description": "Nuclia search widget",
```
Comment (mpellerin42) [inline libs/search-widget/package.json:4]: Maybe we should use the plural form as we provide several widgets, no? (search bar, search results and viewer are distinct widgets)

File: libs/search-widget/package.json [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nuclia/frontend"
+  },
+  "bugs": {
+    "url": "https://github.com/nuclia/frontend/issues"
+  },
+  "peerDependencies": {
+    "@nuclia/core": "^1.1.3",
```
Comment (mpellerin42) [inline libs/search-widget/package.json:21]: you bumped the version the 1.1.4, so I guess we could bump this as well.

File: libs/search-widget/src/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export { default as NucliaSearchBar } from './widgets/search-video-widget/SearchBar.svelte';
+export { default as NucliaSearchResults } from './widgets/search-video-widget/SearchResults.svelte';
+export { default as NucliaViewerWidget } from './widgets/viewer-widget/ViewerWidget.svelte';
```
Comment (mpellerin42) [inline libs/search-widget/src/index.js:3]: might be a good time to rename our widget folder as well as it's not a search-video at all anymore.

File: libs/search-widget/README.md [hunk 4]
```diff
@@ -1,18 +1,42 @@
...
+  zone="europe-1"
+  features="filter,permalink"></nuclia-search-bar>
+
+<nuclia-search-results></nuclia-search-results>
 ```
 
-## Customize style
+## Usage as Svelte components
+
+You will need to install the following dependencies:
```
Comment (mpellerin42) [inline libs/search-widget/README.md:21]: I think you can simplify a bit your sentence: `You need to install…`

File: libs/search-widget/package.json (hunk 2 above)
Comment (ebrehault) [reply libs/search-widget/package.json:21]: ^1.1.3 will get the latest 1.*.* anyway (thanks to `^`)

File: libs/search-widget/src/index.js (hunk 3 above)
Comment (ebrehault) [reply libs/search-widget/src/index.js:3]: maybe later, I am tired refactoring

Comment (ebrehault) [reply libs/search-widget/src/index.js:3]: actually it is needed as `search-widget` is the expected dist folder

**Review by mpellerin42 — APPROVED** [review]

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**